scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,63 +2,31 @@
 
 # fail if some commands fails
 set -e
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
+fi
+if [[ -z $AMQP_URI ]]; then
+    echo "Please set AMQP_URI env with amqp uri or provide path of file containing it as ODO_RABBITMQ_AMQP_URL env"
+    exit 1
+fi
+export JOB_NAME="odo-pr-build"
+export REPO_URL="https://github.com/openshift/odo"
+export TARGET="$(jq .refs.pulls[0].number <<< $(echo $JOB_SPEC))"
```

Comment:
Can you please add a comment here to explain what this is doing? I think it's fetching the PR number and storing it in variable `TARGET`. Also, can I request for a better variable name than `TARGET`, maybe?

Reply:
yep, that is what it is doing. Adding comment.
I think target makes sense here as it couold mean PR no, branch name or tag to `ci-firewall`